<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchByCgpaDialog</class>
 <widget class="QDialog" name="SearchByCgpaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2481</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    color: white;
    font-weight: bold;
    font-size: 14px; 
}


QLineEdit {
    background-color: #333333; 
    color: white;
    border: 1px solid #555555;
    padding: 5px;
}

QTableWidget {
    background-color: #222222; 
    color: white;             
    gridline-color: white;     
    border: 1px solid #555555;
}

QHeaderView::section {
    background-color: #333333; 
    color: white;              
    font-weight: bold;        
    border: 1px solid white;  
    padding: 4px;
}

QPushButton {
    background-color: #333333;
    color: white;
    border: 2px solid #8e24aa; 
    border-radius: 6px;
    padding: 6px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #8e24aa;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>351</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Search Student By CGPA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Exact">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Enter  Student CGPA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Exact"/>
    </item>
    <item>
     <widget class="QLabel" name="label_Min">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Min CGPA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Min"/>
    </item>
    <item>
     <widget class="QLabel" name="label_Max">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Max  CGPA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Max"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Search">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Roll No</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CGPA</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Close">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
